core/txpool/validation.go

```diff
@@ -69,7 +69,7 @@ func ValidateTransaction(tx *types.Transaction, head *types.Header, signer types
 	if tx.Type() == types.DepositTxType {
 		return core.ErrTxTypeNotSupported
 	}
-	if opts.Config.IsOptimism() && tx.Type() == types.BlobTxType {
+	if false && opts.Config.IsOptimism() && tx.Type() == types.BlobTxType {
```

Comment:
Can be put a flag to enable L2 blob rather than false/true (blow)?

Reply:
Done, the flag is named `enable4844`.